src/django/contricleaner/lib/handlers/early_validation_handler.py

```diff
@@ -0,0 +1,28 @@
+from typing import List
+from contricleaner.lib.dto.list_dto import ListDTO
+from contricleaner.lib.handlers.handler import Handler
+from contricleaner.lib.validators.early_validators.\
+    early_header_validator import EarlyHeaderValidator
+from contricleaner.lib.validators.early_validators.\
+    composite_early_validator import CompositeEarlyValidator
+
+
+class EarlyValidationHandler(Handler):
+    __next: Handler
+
+    def setNext(self, next: Handler):
```

Comment:
This function can be deleted from this class since it must be implemented within the parent class.